docs/src/examples/components/Tree/Types/TreeExclusiveExample.shorthand.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { Icon, Tree } from '@stardust-ui/react'
+
+const items = [
+  {
+    key: '1',
+    title: 'one',
+    items: [
+      {
+        key: '2',
+        title: 'one one',
+        items: [
+          {
+            key: '3',
+            title: 'one one one',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    key: '4',
+    title: 'two',
+    items: [
+      {
+        key: '5',
+        title: 'two one',
+      },
+    ],
+  },
+]
+
+const titleRenderer = (Component, { content, open, hasSubtree, ...rest }) => (
+  <Component open={open} {...rest}>
+    {hasSubtree && <Icon name={open ? 'arrow down' : 'arrow right'} />}
+    <span>{content}</span>
+  </Component>
+)
+
+const TreeExclusiveExample = () => <Tree items={items} renderTitle={titleRenderer} />
```

Comment:
could you, pease, suggest what does `exclusive` implies in this example's name? If it is all about custom rendered title, we could consider the following name for this example `TreeTitleCustomizationExample` - this would follow naming conventions we are already using for docs